### PR TITLE
Draft of FAQ answers

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
First draft of FAQ answers. 

@dongbohu would you mind taking a look at the `TODO` under **How can I find out what versions of software/packages were used to process the data?** and leaving your thoughts? Thanks!

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -1,15 +1,48 @@
...
+
 #### How do you process the data?
 
+We process samples run on supported microarray platforms with Single-Channel Array Normalization (SCAN) and transcriptomic samples run on supported sequencing platforms with Salmon and tximport.
+Please see the **Processing Information** section for more details.
+
 #### What type of data does refine.bio support?
 
+refine.bio currently supports gene expression data, specifically genome-scale microarray and RNA-seq data.
+
```
Comment (dvenprasad) [inline docs/faq.md]: Do we have list of supported platforms? Could be useful to link that here.

Comment (jaclyn-taroni) [reply docs/faq.md]: We link to it in the source data section, but I can add the links here as well.

Comment (jaclyn-taroni) [reply docs/faq.md]: I put the links in the question above it because that makes more sense to me, but I can be convinced otherwise.

Comment (dvenprasad) [reply docs/faq.md]: I think the follow-up question that would pop up in the user's brain would be either 'Which ones?' or 'Does it support XYZ thing I'm working with?' and that link would help provide a complete picture.

File: docs/faq.md [hunk 2]
```diff
@@ -1,15 +1,49 @@
...
 #### Why can't I add certain samples to my dataset?
 
+refine.bio will sometimes obtain the metadata (e.g., sample title or experimental protocol) associated with a sample but the raw or submitter processed expression data files are in a format that we can not process. 
+We do not allow you to add these samples to your dataset because we can not deliver expression values.
+
 #### How can I find out what versions of software/packages were used to process the data?
+
+Version information for the packages we think are _most important_ for data processing is available on the pop-up displayed when you click a sample's processing information link.
+
+If you require more granular package information, TODO: how to best access the info in the YAML files?
```
Comment (dongbohu) [inline docs/faq.md]: The YAML file only lists the names of the packages that we want to keep track of. I think the best way to get the processing information is using the API directly, like:
- All processors: https://api.staging.refine.bio/processors/ 
- Per sample: https://api.staging.refine.bio/samples/1/ (see the `results` section`)

Right now the second API only lists the processor's ID, but we can modify it to show the full information of that processor. 

Does this make sense?

**Comment by jaclyn-taroni** [comment]
Yep, thanks @dongbohu I've filed #27

File: docs/faq.md [hunk 3]
```diff
@@ -44,6 +44,10 @@ We do not allow you to add these samples to your dataset because we can not deli
 
 Version information for the packages we think are _most important_ for data processing is available on the pop-up displayed when you click a sample's processing information link.
```
Comment (dvenprasad) [inline docs/faq.md]: Screenshot here?

Comment (jaclyn-taroni) [reply docs/faq.md]: @dvenprasad check out the most recent commit and let me know what you think of the sizing

Comment (dvenprasad) [reply docs/faq.md]: 👍

**Comment by jaclyn-taroni** [comment]
I'm going to merge this as is. If we want to make tweaks to the API processor list bit, we can address that at the same time as #27.

**Pull request merged**